src/elm/Page/Dashboard/Claim.elm

```diff
@@ -87,7 +93,7 @@ view ({ shared } as loggedIn) model =
                         Page.fullPageLoading
 
                     Loaded claim ->
-                        div [ class "bg-white py-2" ]
+                        div [ class "bg-gray-100 py-2" ]
```

Comment:
this causes a very little very subtle change on the background:
<img width="288" alt="Screen Shot 2020-10-20 at 18 33 23" src="https://user-images.githubusercontent.com/1082127/96646641-cf142b80-1302-11eb-863a-c2d5754e8c3c.png">


I just removed it and it looked good!

Reply:
Yep, this change was intentional. The [Figma design for the updated Claim View page](https://www.figma.com/file/N2lbB9xdO6tFbfg2Is4h7s/claim_with_image?node-id=2%3A1309) has a gray background.
![image](https://user-images.githubusercontent.com/140053/96677779-f6a5db00-1378-11eb-9d85-ba7682eb3f11.png)

Reply:
oh i see!